test/server_integration/http/ssl_redirect/index.ts

```diff
@@ -6,19 +6,23 @@
  * Side Public License, v 1.
  */
 
-export default function ({ getService }) {
+import { FtrProviderContext } from '../../services/types';
+
+export default function ({ getService }: FtrProviderContext) {
   const supertest = getService('supertest');
 
-  // Failing: See https://github.com/elastic/kibana/issues/131192
-  // Failing: See https://github.com/elastic/kibana/issues/131192
-  describe.skip('kibana server with ssl', () => {
+  describe('kibana server with ssl', () => {
     it('redirects http requests at redirect port to https', async () => {
       const host = process.env.TEST_KIBANA_HOST || 'localhost';
       const port = process.env.TEST_KIBANA_PORT || '5620';
       const url = `https://${host}:${port}/`;
 
       await supertest.get('/').expect('location', url).expect(302);
+    });
 
+    // Skips because the current version of supertest cannot follow redirects
+    // Can be unskipped once https://github.com/elastic/kibana/pull/163716 is merged
+    it.skip('does not boot-loop (2nd redirect points to the landing page)', async () => {
```

Comment:
Being pedantically explicit, this 2nd check didn't test what the previous claimed (redirecting to the https port). That's why I split it to a different test.

Then, I found out the issue with the old library of `supertest`, and I had to skip it.